cmake/Modules/info.cmake

```diff
@@ -44,5 +44,3 @@ else()
   message(STATUS "- DBus Remote Driver (disabled)")
 endif()
 message(STATUS " ")
-message(STATUS "DEFINES = ${DEFINES}")
-message(STATUS " ")
```

Comment:
Do you know why this is not showing anything anymore? It was quite useful before, when it was showing all the defines.

Is that maybe: https://cmake.org/cmake/help/latest/prop_dir/COMPILE_DEFINITIONS.html ?

Reply:
Oops, I removed that because I didn't remember it working before, figured it was just always messed up or been that way a long time.  I just realized that its actually because I moved the properties from being directory-based to target-based when I was cleaning things up for mimalloc.  The `DEFINES` variable gets set on the first line of `info.cmake`, and is used in the various input driver conditionals as well.

Fixed it just now and force pushed.